README.md

```diff
@@ -2,17 +2,17 @@
   <img src="https://upper.io/db.v3/images/gopher.svg" width="256" />
 </p>
 
-# upper.io/db.v3 [![Build Status](https://travis-ci.org/upper/db.svg?branch=master)](https://travis-ci.org/upper/db) [![GoDoc](https://godoc.org/upper.io/db.v3?status.svg)](https://godoc.org/upper.io/db.v3)
+# upper/db [![Build Status](https://travis-ci.org/upper/db.svg?branch=v4)](https://travis-ci.org/upper/db) [![GoDoc](https://godoc.org/github.com/upper/db?status.svg)](https://godoc.org/github.com/upper/db)
 
-The `upper.io/db.v3` package for [Go][2] is a productive data access layer for
-Go that provides a common interface to work with different data sources such as
+`upper/db` is a productive data access layer (DAL) for [Go](https://golang.org)
+that provides agnostic tools to work with different data sources, such as
 [PostgreSQL](https://upper.io/db.v3/postgresql),
 [MySQL](https://upper.io/db.v3/mysql), [SQLite](https://upper.io/db.v3/sqlite),
-[MSSQL](https://upper.io/db.v3/mssql),
-[QL](https://upper.io/db.v3/ql) and [MongoDB](https://upper.io/db.v3/mongo).
+[MSSQL](https://upper.io/db.v3/mssql), [QL](https://upper.io/db.v3/ql) and
+[MongoDB](https://upper.io/db.v3/mongo).
 
 ```
-go get upper.io/db.v3
+go get github.com/upper/db
```

Comment:
if you include a go.mod file, you'll prob want to specify the module as github.com/upper/db/v4